website/static/js/pages/sharing-page.js

```diff
@@ -40,3 +88,48 @@ $(function() {
         trigger: 'hover'
     });
 });
+
+var checkWindowWidth = function() {
+    if ($(window).width() <= 600) {
+        $('table.responsive-table-xxs tbody tr td:first-child').attr('role','button').attr('onclick', 'toggleExpand(this.parentElement)');
```

Comment:
It represents different points at which the tables collapse to the alternate view.  It is currently set up as 'responsive-table responsive-table-xxs' and 'responsive-table responsive-table-xs'